### PR TITLE
Add changelog for script/Compiler/IDE

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
Still missing some changelogs for Compiler and IDE.

### CODE REVIEW

**Review by garyverhaegen-da — APPROVED** [review]
I'm not sure exactly who or what looks at this file, so please make sure the [release document](https://docs.google.com/document/d/1FaBFuYweYt0hx6fVg9rhufCtDNPiATXu5zwN2NWp2s4/edit#bookmark=id.227zecfjkiix) is also updated accordingly.

File: UNRELEASED.md [hunk 1]
```diff
@@ -68,3 +68,47 @@ The following metrics were added:
...
+    number of get time calls made in a rule evaluation
+    number of active and pending contracts at start and end of a rule evaluation
+    number of in-flight commands at start and end of a rule evaluation
+    size of a message batch for rule evaluation (with a breakdown by number of completion successes, completion failures, transaction creates, transaction archives and heartbeats)
+
+    trigger TRACE logging now logs the full trigger state and the full message batch (this can be very noisy)
+
+#### Bug fixes:
+
+    release logback.xml now logs context and trigger logging is INFO by default
```
Comment (basvangijzel-DA) [inline UNRELEASED.md:98]: What do you mean with context? stacktrace, some of the relevant variables?

**Review by basvangijzel-DA — APPROVED** [review]

File: UNRELEASED.md (hunk 1 above)
Comment (remyhaemmerle-da) [reply UNRELEASED.md:98]: cc @carlpulley-da

Comment (carlpulley-da) [inline UNRELEASED.md:98]:
```suggestion
    release logback.xml now logs context and trigger metrics logging is INFO by default
```

Comment (carlpulley-da) [reply UNRELEASED.md:98]: Previously, trigger production logback.xml files never included the logging context

File: UNRELEASED.md [hunk 2]
```diff
@@ -68,3 +68,47 @@ The following metrics were added:
...
+    potential trigger OOM issue in converter
+    trigger context now correctly logged with Daml calls to debugRaw
```
[the remark this answers is not in the document]
Comment (basvangijzel-DA) [reply UNRELEASED.md:100]: Thanks for the comments Carl, I've adapted it in the release notes!

**Pull request closed without merging**